src/components/SignIn/SignInForm.js

```diff
@@ -0,0 +1,146 @@
+import Button from '@material-ui/core/Button';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import FormControl from '@material-ui/core/FormControl';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import React, { useCallback, useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import * as ROUTES from '../../constants/routes';
+
+const RoundedTextField = withStyles((theme) => ({
+  root: {
+    '& fieldset': {
+      borderRadius: 30,
+    },
+    '& input': {
+      marginLeft: theme.spacing(1),
+      marginRight: theme.spacing(1),
+    },
+  },
+}))(TextField);
+
+const RoundedButton = withStyles((theme) => ({
+  root: {
+    borderRadius: 30,
+    height: theme.typography.fontSize * 3,
+    margin: theme.spacing(3, 0, 2),
+    textTransform: 'none',
+    width: '50%',
+  },
+}))(Button);
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  buttonWrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    position: 'relative',
+  },
```

Comment:
It looks like with lower screen/browser widths the text in the button overflows, but the button doesn't expand to meet the text, maybe this could be fixed with css?

Reply:
It can definitely be fixed with better styling.